scripts/install-c-aci-testing.sh

```diff
@@ -2,5 +2,5 @@
 
 latest=$(gh release list -R microsoft/confidential-aci-testing -L 1 --json tagName --jq '.[0].tagName')
 gh release download $latest -R microsoft/confidential-aci-testing
-pip install c-aci-testing*.tar.gz
+python -m pip install c-aci-testing*.tar.gz
```

Comment:
Rather than using the setup script to call this script, can we just merge this script into the setup script? Seems part of setup anyway

Reply:
I think I've mentioned in another reply, but setting up the testing package is the only thing we need in CI, but for local development we need more which is what setup.sh is for, so I think keeping them separate makes sense, also it's handy if I release a new version of the testing package to have a target just for that